### PR TITLE
upload_images

### PR DESCRIPTION
**Pull request by nguyenvanson332000**
## Checklist
- [ ] This is bug/feature (delete incorrect one)
- [ ] Title including ticket number and title

## What this PR do?
Explain clearly and briefly what this PR does. Make a meaningful sentence instead of "this PR solve ticket #xxx"

## Context/Background
More information on the issue that you want to provide, related issues and PRs you have to write information to ask orally or in informal place too.

## Screenshots / Demo & Instruction for testing (if applicable)
image
image
image

## References
- Blog posts / web pages relevant to this issue (which you get your solution / idea from and it may help others to understand the issue quickly)
- Do we need update our knowledge base / project wiki?

### CODE REVIEW

File: app/views/admin/exams/_form.slim [hunk 1]
```diff
@@ -1,6 +1,9 @@
 = form_for([:admin, @exam]) do |l|
```
Comment (rikai-hai) [inline app/views/admin/exams/_form.slim:1]:
```suggestion
= form_for([:admin, @exam]) do |f|
```

File: app/views/admin/exams/_form.slim [hunk 2]
```diff
@@ -1,6 +1,9 @@
...
   fieldset
     = render "shared/error_messages", object: l.object
+    = l.label t(".picture")
+    = l.fields_for :image_attributes do |image_l|
```
Comment (rikai-hai) [inline app/views/admin/exams/_form.slim:5]:
```suggestion
    = l.fields_for :image_attributes do |ff|
```

File: app/views/users/edit.slim [hunk 3]
```diff
@@ -2,11 +2,17 @@
...
-    .gravatar_edit
+    - if @user.image.blank?
+      .gravatar_edit
         = gravatar_for@user
+    - else
+      .avatar_edit
+        img.avatar-user alt="#{@user.name}" src="#{@user.image.image_url}"
     .update_user
-      = form_for(@user) do |f|
+      = form_for @user, :html => { :multipart => true } do |f|
```
Comment (rikai-hai) [inline app/views/users/edit.slim:12]:
```suggestion
      = form_for @user, html: { multipart: true } do |f|
```

File: app/views/users/edit.slim [hunk 4]
```diff
@@ -2,11 +2,17 @@
...
         = render'shared/error_messages',object: @user
+        = f.fields_for :image_attributes do |image_f|
+          = image_f.file_field :image_url, class: "input-file previewIMG"
```
Comment (rikai-hai) [inline app/views/users/edit.slim:15]: class kiểu gì đấy

File: app/views/admin/exams/_form.slim (hunk 1 above)
Comment (nguyenvanson332000) [reply app/views/admin/exams/_form.slim:1]: dạ oke anh

File: app/views/users/edit.slim (hunk 4 above)
Comment (nguyenvanson332000) [reply app/views/users/edit.slim:15]: à dạ bữa em xóa css nó mà em quên xóa nó ạ để em xóa nó đi

**Pull request merged**